### PR TITLE
docs: assets: gallery: Added current fundraiser image

### PR DESCRIPTION
**Pull request by okawo80085**
Added current fundraiser post

### CODE REVIEW

File: docs/_posts/2024-03-10-fundraiser-obbp-13.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+![obbp13]({{ "assets/images/gallery/obbp13.png" | relative_url }})
+
+{% include donate_button.html %}
+
+In December of 2023 we opened `Workshop №0` in Lviv led by [@SpiderPup](https://twitter.com/SpiderPup69)
+
+We are a international team of volunteers based in Lviv, Ukraine. Dedicated to supporting drone teams, we raise funds and craft cutting-edge equipment, from electronics to radio kits, drones, drop rigs, fins, and `[redacted]`. Brace for more boom 💥
+
```
Comment (NAFOArt) [inline docs/_posts/2024-03-10-fundraiser-obbp-13.md]: We are **an** international team

File: docs/assets/css/nafoarms.scss [hunk 2]
```diff
@@ -124,6 +124,12 @@ body {
     padding-bottom: 2vw;
 }
 
+.redacted-tag {
+    background-color: balck;
```
Comment (NAFOArt) [inline docs/assets/css/nafoarms.scss]: Black has a typo

**Review by NAFOArt — CHANGES_REQUESTED** [review]
See individual line comments. The a/an typo is unimportant but the CSS typo should probably be fixed

File: docs/assets/css/nafoarms.scss (hunk 2 above)
Comment (okawo80085) [reply docs/assets/css/nafoarms.scss]: Oh actually this is from an unfinished thing, forgot to remove it

File: docs/_posts/2024-03-10-fundraiser-obbp-13.md (hunk 1 above)
Comment (okawo80085) [reply docs/_posts/2024-03-10-fundraiser-obbp-13.md]: Yup gimme a sec

Comment (okawo80085) [reply docs/_posts/2024-03-10-fundraiser-obbp-13.md]: should be fixed now

File: docs/assets/css/nafoarms.scss (hunk 2 above)
Comment (okawo80085) [reply docs/assets/css/nafoarms.scss]: should be fixed now

**Review by NAFOArt — DISMISSED** [review]

**Comment by okawo80085** [comment]
Dang github's merge resolver dropped the review xd

**Review by NAFOArt — APPROVED** [review]
IMG_7438

**Pull request merged**